examples/mysql-wordpress-pd/README.md

```diff
@@ -34,301 +34,231 @@ Documentation for other releases can be found at
+trailing newline.
+
+```shell
+kubectl create -f https://raw.githubusercontent.com/kubernetes/kubernetes/master/examples/mysql-wordpress-pd/local-volumes.yaml
+kubectl create secret generic mysql-pass --from-file=password.txt
+kubectl create -f https://raw.githubusercontent.com/kubernetes/kubernetes/master/examples/mysql-wordpress-pd/mysql-deployment.yaml
+kubectl create -f https://raw.githubusercontent.com/kubernetes/kubernetes/master/examples/mysql-wordpress-pd/wordpress-deployment.yaml
 ```
 
-Next, start up a Kubernetes cluster:
-
-```sh
-wget -q -O - https://get.k8s.io | bash
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+##Table of Contents
+
+  - [Cluster Requirements](#cluster-requirements)
+  - [Decide where you will store your data](#decide-where-you-will-store-your-data)
+    - [GCE Persistent Disk](#gce-persistent-disk)
+    - [Host Path](#host-path)
+  - [Create the MySQL Password Secret](#create-the-mysql-password-secret)
+  - [Deploy MySQL](#deploy-mysql)
+  - [Deploy WordPress](#deploy-wordpress)
+  - [Visit your new WordPress blog](#visit-your-new-wordpress-blog)
+  - [Take down and restart your blog](#take-down-and-restart-your-blog)
+  - [Next Steps](#next-steps)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Cluster Requirements
+
+Consult a
+[Getting Started Guide](../../docs/getting-started-guides/)
+to set up a cluster and the kubectl command line client.
+
+Kubernetes runs in a variety of environments and is inherently
+modular. Not all clusters are the same. These are the requirements for
+this example.
+
+* Kubernetes version 1.2 is required due to using newer features, such
+  at PV Claims and Deployments.
+* [Cluster DNS](../../cluster/addons/dns/) will be used for service discovery.
+* An [external load balancer](../../docs/user-guide/services.md#type-loadbalancer)
+  will be used to access WordPress.
+* [Persistent Volume Claims](../../docs/user-guide/persistent-volumes.md)
+  are used. You must create Persistent Volumes in your cluster to be
+  claimed. This example demonstrates how to create two types of
+  volumes, but any volume is sufficient.
+
+## Decide where you will store your data
+
+Both MySQL and WordPress will have a
+[Persistent Volume](../../docs/user-guide/persistent-volumes.md) to
+store their data. We will use a Persistent Volume Claim to claim an
+available persistent volume. This example covers HostPath and
+GCEPersistentDisk volumes. Choose one of the two, or see
+[Types of Persistent Volumes](../../docs/user-guide/persistent-volumes.md#types-of-persistent-volumes)
+for more options.
+
+### GCE Persistent Disk
+
+Please see the
+[GCE getting started guide](../../docs/getting-started-guides/gce.md)
+for full details and other options for starting a cluster.
+
+Create two persistent disks. You will need to create the disks in the
```

Comment:
k8s v1.2 adds support dynamic provisioning support (https://docs.openshift.org/latest/install_config/persistent_storage/dynamically_provisioning_pvs.html#dynamic-pvs-requesting-storage), so that volumes don't need to be manually provisioned ahead of time--but it's still experimental. After v1.3 is launched, we should update this example to use dynamic provisioning.